tests/io/test_RiverParameterData.py

```diff
@@ -0,0 +1,556 @@
+import sys
+from contextlib import contextmanager
+from io import StringIO
+import pytest
+import configparser
+
+from dfastmi.io.RiverParameterData import RiverParameterData
+
+@contextmanager
+def captured_output():
+    new_out, new_err = StringIO(), StringIO()
+    old_out, old_err = sys.stdout, sys.stderr
+    try:
+        sys.stdout, sys.stderr = new_out, new_err
+        yield sys.stdout, sys.stderr
+    finally:
+        sys.stdout, sys.stderr = old_out, old_err
+
+def _initialize_river_data_from_config(config):
+        branches = ["Branch1", "Branch2"]
+        nreaches = [2, 3]
+        river_data = RiverParameterData(config)
+        river_data.initialize(branches, nreaches)
+        return river_data
+
+class Test_collect_values1():
+    
+    def test_collect_values1_01(self):
```

Comment:
should we keep the names of theses tests as this?

Reply:
Refactoring & improvements should be done in separate issue for each component (unit) test files should be done too

Reply:
https://issuetracker.deltares.nl/browse/DFAST-112